flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-kafka/src/main/java/org/apache/flink/cdc/connectors/kafka/sink/KafkaDataSinkFactory.java

```diff
@@ -72,9 +74,15 @@ public DataSink createDataSink(Context context) {
                 KeySerializationFactory.createSerializationSchema(configuration, keyFormat, zoneId);
         JsonSerializationType jsonSerializationType =
                 context.getFactoryConfiguration().get(KafkaDataSinkOptions.VALUE_FORMAT);
+        Boolean isIncludedDebeziumSchema =
+                context.getFactoryConfiguration().get(SINK_DEBEZIUM_JSON_SCHEMA_ENABLED);
+        Preconditions.checkArgument(
+                !(isIncludedDebeziumSchema
+                        && !jsonSerializationType.equals(JsonSerializationType.DEBEZIUM_JSON)),
+                "sink.debezium_json_schema.enabled only supported debezium-json.");
```

Comment:
is only supported  when using debezium-json.

Reply:
We can move the check to ChangeLogJsonFormatFactory.createSerializationSchema method.